mxcubecore/model/lims_session.py

```diff
@@ -306,3 +306,22 @@ class SampleInformation(BaseModel):
     createdAt: datetime
     updatedAt: datetime
     resources: List[Resource]  # List of associated resources
+
+
+class Download(BaseModel):
+    """
+    Represents a downloadable file with metadata used to match pipeline references.
+
+    Parameters:
+        path (str): Absolute file path to the downloaded file on the filesystem.
+        filename (str): Name of the file, used for matching with pipeline references.
+        groupName (Optional[str]): Optional group identifier, used to match with
+        search models.
+        Can be None or a string such as 'Group1', 'Group2', etc.
+    """
```

Comment:
Skip the types in docstrings if there are already type hint annotations in the code.
Also check how it renders: <https://mxcubecore--1260.org.readthedocs.build/en/1260/dev/autosummary/mxcubecore.model.lims_session.html#mxcubecore.model.lims_session.Download>.

```suggestion
    """
    Represents a downloadable file with metadata used to match pipeline references.

    Parameters:
        path: Absolute file path to the downloaded file on the filesystem.
        filename: Name of the file, used for matching with pipeline references.
        groupName:
            Optional group identifier, used to match with search models.
            Can be None or a string such as 'Group1', 'Group2', etc.
    """
```